lib/active_record/connection_adapters/odbc_adapter.rb

```diff
@@ -1368,7 +1359,7 @@ def insert_sql(sql, name = nil, pk = nil, id_value = nil, sequence_name = nil) #
           retry_count = 0
           begin
             pre_insert(sql, name, pk, id_value, sequence_name) if respond_to?("pre_insert")
-            stmt = @connection.run(sql)
+            stmt = log(sql, name) { @connection.run(sql) }
```

Comment:
Actually it is, this is another one of the changes that I pulled in from the localytics-rails local version.